modules/api/src/main/java/org/apache/fluo/api/observer/StringObserver.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.fluo.api.observer;
+
+import org.apache.fluo.api.client.TransactionBase;
+import org.apache.fluo.api.data.Bytes;
+import org.apache.fluo.api.data.Column;
+
+/**
+ * @since 1.1.0
+ */
+@FunctionalInterface
+public interface StringObserver extends Observer {
+
+  @Override
+  default void process(TransactionBase tx, Bytes row, Column col) throws Exception {
+    process(tx, row.toString(), col);
```

Comment:
Not sure it matters much, since it's API, you could have a helpful message here instead of a generic NPE:
you could do `Objects.requireNonNull(row, "message").toString()` instead of `row.toString()`

Also, I'm guessing `Bytes#toString()` assumes `UTF-8`? If not, this could have unpredictable results.